### PR TITLE
fix(service): correctly set project_name slug on project create

### PR DESCRIPTION
**Pull request by jsam**

### CODE REVIEW

File: renku/core/models/git.py [hunk 1]
```diff
@@ -105,13 +105,16 @@ def __attrs_post_init__(self):
 
     @classmethod
     def parse(cls, href):
-        """Derive basic informations."""
+        """Derive URI components."""
+        if href.isascii() is False:
```
Comment (mohammad-sdsc) [inline renku/core/models/git.py:109]: Is there is reason for not using `if not href.isascii()`?

File: renku/core/utils/scm.py [hunk 2]
```diff
@@ -19,9 +19,19 @@
 import re
 
 
-def strip_and_lower(input):
+def normalize_to_ascii(input_string, sep="-"):
     """Adjust chars to make the input compatible as scm source."""
-    return re.sub(r"\s", r"-", input.strip()).lower()
+    return (
+        f"{sep}".join(
```
Comment (mohammad-sdsc) [inline renku/core/utils/scm.py:25]: sep is a string so `f"{sep}"` can be replaced with `sep`.

**Review by mohammad-sdsc — APPROVED** [review]
LGTM, thanks! I've made two minor comments, feel free to ignore!

**Review by mohammad-sdsc — APPROVED** [review]
🚀

**Comment by lorenzo-cavazzi** [comment]
This comes late but the logic for deriving slugs is still slightly different from the GitLab.
Specifically, multiple separators `-` should be collapsed.

You can verify this in the preview here: https://lorenzo.dev.renku.ch/projects/new
A title like `test --üäüaua` should create a slug like `test-aua` instead of `test----aua`

**Pull request merged**